cmd/delete/cmd.go

```diff
@@ -0,0 +1,155 @@
+package uninstall
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/avast/retry-go"
+	"github.com/pkg/errors"
+
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/kyma-project/cli/pkg/asyncui"
+	"github.com/spf13/cobra"
+
+	installConfig "github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/deployment"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/helm"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/overrides"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new kyma command
+func NewCmd(o *Options) *cobra.Command {
+
+	cmd := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "delete",
+		Short:   "Deletes Kyma from a running Kubernetes cluster.",
+		Long:    `Use this command to delete Kyma from a running Kubernetes cluster.`,
+		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.Run() },
+		Aliases: []string{"d"},
+	}
+
+	cobraCmd.Flags().DurationVarP(&o.Timeout, "timeout", "", 1200*time.Second, "Maximum time for the deletion")
+	cobraCmd.Flags().DurationVarP(&o.TimeoutComponent, "timeout-component", "", 360*time.Second, "Maximum time to delete the component")
+	cobraCmd.Flags().IntVar(&o.Concurrency, "concurrency", 4, "Number of parallel processes")
+	cobraCmd.Flags().BoolVarP(&o.KeepCRDs, "keep-crds", "", false, "Flag specifying whether to keep CRDs on deletion")
```

Comment:
We should rephrase this similar to `deploy --atomic`
 
```suggestion
	cobraCmd.Flags().BoolVarP(&o.KeepCRDs, "keep-crds", "", false, "Set --keep-crds=true to keep CRDs on deletion")
```